specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingProperty.json

```diff
@@ -65,182 +95,553 @@
+          "x-ms-enum": {
+            "name": "AccountSubType",
+            "modelAsString": true
+          }
+        },
+        "billingCurrency": {
+          "description": "The billing currency for the subscription. Available for billing accounts with agreement type Enterprise Agreement",
+          "type": "string",
+          "readOnly": true
+        },
+        "billingProfileDisplayName": {
+          "description": "The name of the billing profile for the invoice section or customer.",
+          "type": "string",
+          "readOnly": true
+        },
+        "billingProfileId": {
+          "description": "The fully qualified ID that uniquely identifies a billing profile.",
+          "type": "string",
+          "readOnly": true
         },
         "billingProfileSpendingLimit": {
           "description": "The billing profile spending limit.",
           "enum": [
             "Off",
             "On"
           ],
-          "readOnly": true,
           "type": "string",
+          "readOnly": true,
           "x-ms-enum": {
-            "modelAsString": true,
-            "name": "BillingProfileSpendingLimit"
+            "name": "SpendingLimit",
+            "modelAsString": true
           }
         },
+        "billingProfileSpendingLimitDetails": {
+          "description": "Identifies the billing profile that is linked to another billing profile in indirect purchase motion.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SpendingLimitDetails"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": []
+        },
         "billingProfileStatus": {
           "description": "The status of the billing profile.",
           "enum": [
+            "Unknown",
             "Active",
             "Disabled",
-            "Warned"
+            "Warned",
+            "Deleted",
+            "UnderReview"
           ],
-          "readOnly": true,
           "type": "string",
+          "readOnly": true,
           "x-ms-enum": {
-            "modelAsString": true,
-            "name": "BillingProfileStatus"
+            "name": "BillingProfileStatus",
+            "modelAsString": true
           }
         },
         "billingProfileStatusReasonCode": {
           "description": "Reason for the specified billing profile status.",
           "enum": [
             "PastDue",
+            "UnusualActivity",
             "SpendingLimitReached",
             "SpendingLimitExpired"
           ],
+          "type": "string",
           "readOnly": true,
+          "x-ms-enum": {
+            "name": "BillingProfileStatusReasonCode",
+            "modelAsString": true
+          }
+        },
+        "billingProfilePaymentMethodType": {
+          "description": "The payment method type of the primary payment method for the billing profile.",
+          "enum": [
+            "Unknown",
+            "Credits",
+            "Creditcard",
+            "Checkwire"
```

Comment:
Can we cross-check with our PaymentMethods API if these values are identical? I believe they may have slightly different casing...